src/sec_related_rates.mbx

```diff
@@ -269,232 +269,303 @@
+              <set>
+                <member correct="yes">True</member>
+                <member>False</member>
+              </set>
+            </var>
+            <var name="$false">
+              <set>
+                <member>True</member>
+                <member correct="yes">False</member>
+              </set>
             </var>
             <pg-code>
+              $true=PopUp(['?','True','False'],1);
+              $false=PopUp(['?','True','False'],2);
             </pg-code>
           </setup>
           <statement>
-            <p>T/F: Implicit differentiation is often used when solving <q>related rates</q> type problems.</p>
+            <p>T/F: Implicit differentiation is often used when solving <q>related rates</q> type problems. <var name="$true" form="popup"/></p>
           </statement>
-          <solution>
-            <p>T</p>
-          </solution>
         </webwork>
       </exercise>
+<!--Exercise 2: Bizarre T/F-->
       <exercise>
         <webwork seed="1">
           <setup>
-            <var name="">
-              <static></static>
+             <var name="$true">
+              <set>
+                <member correct="yes">True</member>
+                <member>False</member>
+              </set>
+            </var>
+            <var name="$false">
+              <set>
+                <member>True</member>
+                <member correct="yes">False</member>
+              </set>
             </var>
             <pg-code>
+              $true=PopUp(['?','True','False'],1);
+              $false=PopUp(['?','True','False'],2);
             </pg-code>
           </setup>
           <statement>
-            <p>T/F: A study of related rates is part of the standard police officer training.</p>
+            <p>T/F: A study of related rates is part of the standard police officer training. <var name="$false" form="popup"/></p>
           </statement>
-          <solution>
-            <p>F</p>
-          </solution>
         </webwork>
       </exercise>
     </exercisegroup>
+
+<!--Group 2: Related rates answer entry-->
     <exercisegroup>
       <title>Problems</title>
+<!--Exercise 3: Puddle of depth 10 cm growing-->
       <exercise>
         <webwork seed="1">
           <setup>
-            <var name="">
-              <static></static>
+            <var name="$a">
+              <static>0.7958</static>
+            </var>
+            <var name="$b">
+              <static>0.07958</static>
+            </var>
+            <var name="$c">
+              <static>0.007958</static>
             </var>
             <pg-code>
+              <!--FUNGUS: Is this correct?-->
+              $a = 0.7958;
+              $b = 0.07958;
+              $c = 0.007958;
             </pg-code>
           </setup>
           <statement>
             <p>Water flows onto a flat surface at a rate of 5cm<m>^3</m>/s forming a circular puddle 10mm deep. How fast is the radius growing when the radius is:</p>
```

Comment:
I'll follow that guideline.

The exact code you put in renders as '5cc7' instead of as intended. Probably something minor, but I don't know the fix.